mod_modular_vanilla/hooks/ui/screens/tactical/modules/turn_sequence_bar/turn_sequence_bar.nut

```diff
@@ -1,5 +1,25 @@
 ::ModularVanilla.QueueBucket.VeryLate.push(function() {
 	::ModularVanilla.MH.hook("scripts/ui/screens/tactical/modules/turn_sequence_bar/turn_sequence_bar", function(q) {
+		q.m.__MV_FirstSlotEntityID <- null;
```

Comment:
We also need to set this value to `null` at the end of every characters turn.
Otherwise we don't get a `onResumeTurn`, if 
- the only character left to act in the current turn, waits
- the last character to act in a turn is also the first character to act in the next turn

Both cause the same EntityID to legitimately enter the first slot twice in a row